code/game/objects/items/weapons/clown_items.dm

```diff
@@ -165,6 +165,26 @@
 					continue
 			M.emote("flip")
 
+/obj/item/weapon/bikehorn/saxophone

```

Comment:
Yeah, so if you do attack_self or Crossed() it should use that sound, so you only need to change it in one place rather than 2+